classes/phing/tasks/ext/SymlinkTask.php

```diff
@@ -236,7 +236,36 @@ protected function getMap()
             }
 
             if ($this->isRelative()) {
-                $fromDir = $fs->getDir($this->getProject())->getPath();
+                $endPath = $fs->getDir($this->getProject())->getAbsolutePath();
+                $startPath = $this->getLink();
```

Comment:
This only works if the link path given is absolute, how to get absolute path from `$this->getLink()?